Makefile

```diff
@@ -1,7 +1,7 @@
 ROOT_DIR := .
 SERVER_EXECUTEBLE := "$(ROOT_DIR)/pkg/cmd/server/server"
 
-OVN_KUBERNETES_ROOT ?= $(ROOT_DIR)/../../ovn-org/ovn-kubernetes
+OVN_KUBERNETES_ROOT ?= $HOME/work/src/github.com/ovn-org/ovn-kubernetes
```

Comment:
Why did you replace it? it is wrong.

Reply:
as we talked earlier I replace `OVN_KUBERNETES_ROOT `  to match the path on [OVN kubernetes](https://github.com/ovn-org/ovn-kubernetes). any way , I revert this change.